images.json

```diff
@@ -85,6 +85,27 @@
       }
     },
 
+    "debootstrap" : {
+        "name" : "Debootstrap entry (to be used scripts/make-s2e-image.sh only)",
```

Comment:
"to be used *with*"

Reply:
I wonder what would happen if you run `s2e image_build debootstrap`.
Given that it's very different from the existing build infrastructure, I'd probably add separate support for it instead of adding an artificial entry in image.json.